Examples/MAX32520/AES/main.c

```diff
@@ -416,8 +416,10 @@ int main(void)
 
     if (fail == 0) {
         printf("\nExample Succeeded\n");
+        return 0;
     } else {
         printf("\nExample Failed\n");
+        return -1;
     }
 
     return 0;
```

Comment:
This can be removed, it'll never reach here.

Reply:
@sihyung-maxim @ozersa I will remove such unnecessary return values on the examples modified in this branch. I will merge it after such change.